releasenotes/notes/enable-istio-tls-by-default-1da65945911f278a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Enable USM Istio TLS monitoring by default, if USM is enabled.
```

Comment:
Considering the underlying config is not part of the config template (so not "known" by users), I think the release note should either:
- not say "by default" (and maybe say "Implement" instead of "Enable" ?) 
- explain how to disable it (and in that case the config key should be added to the template)